plugins/outputs/influxdb/http.go

```diff
@@ -209,13 +209,19 @@ func (c *httpClient) CreateDatabase(ctx context.Context, database string) error
 	}
 	defer resp.Body.Close()
 
+	// Track that a 'create database` statement was executed for the database
+	// even if the statement fails.  One initial attempt to create the database
+	// is made each time a new database is encountered in the database_tag.
+	// After this initial attempt it will only be executed if a "database not
+	// found" error occurs.
+	c.createDatabaseExecuted[database] = true
```

Comment:
seems like we should only do this on success or permission errors. There's some errors communicating with the database that shouldn't trigger it to stop trying.